lib/src/test/kotlin/starknet/account/StandardAccountTest.kt

```diff
@@ -343,21 +365,24 @@ class StandardAccountTest {
             // docsEnd
             assertEquals(TransactionVersion.V3_QUERY, declareTransactionPayload.version)
             // docsStart
-            val request = provider.getEstimateFee(payload = listOf(declareTransactionPayload), simulationFlags = emptySet())
+            val request =
+                provider.getEstimateFee(payload = listOf(declareTransactionPayload), simulationFlags = emptySet())
             val feeEstimate = request.send().values.first()
             // docsEnd
             assertNotEquals(Felt.ZERO, feeEstimate.overallFee)
             assertEquals(
-                feeEstimate.gasPrice.value * feeEstimate.gasConsumed.value + feeEstimate.dataGasPrice.value * feeEstimate.dataGasConsumed.value,
+                feeEstimate.l1GasPrice.value * feeEstimate.l1GasConsumed.value + feeEstimate.l1DataGasPrice.value * feeEstimate.l1DataGasConsumed.value + feeEstimate.l2GasPrice.value * feeEstimate.l2GasConsumed.value,
                 feeEstimate.overallFee.value,
             )
         }
     }
 
     @Test
     fun estimateMessageFee() {
-        val l1l2ContractCode = Path.of("src/test/resources/contracts_v2/target/release/ContractsV2_l1_l2.sierra.json").readText()
-        val l1l2CasmContractCode = Path.of("src/test/resources/contracts_v2/target/release/ContractsV2_l1_l2.casm.json").readText()
+        val l1l2ContractCode =
+            Path.of("src/test/resources/contracts_v2/target/release/ContractsV2_l1_l2.sierra.json").readText()
+        val l1l2CasmContractCode =
+            Path.of("src/test/resources/contracts_v2/target/release/ContractsV2_l1_l2.casm.json").readText()
```

Comment:
Just to clarify, are these changes necessary (e.g. dictated by linting)? 👀

Reply:
Oops, not needed. Removed 🫡 .